sources/packages/forms/src/form-definitions/partnerinformationandincomeappeal.json

```diff
@@ -1689,7 +1689,8 @@
               "dir": "Current Year Income - Supporting Documents",
               "customClass": "font-weight-bold",
               "validate": {
-                "required": true
+                "required": false,
+                "custom": "const maxUploadedFiles = 25; const files = data[instance.key] || []; valid = !files.length ? 'At least one file is required' : files.length > maxUploadedFiles ? `Maximum ${maxUploadedFiles} files allowed` : true;"
```

Comment:
The `maxUploadedFiles` can be a hidden(calculated) variable injected into the form from Vue and API and used so that we can centralize the config. If there is a considerable amount of changes to achieve that, then at bare-minimum there must be hidden variable (calculated) assigned with value as 25 at each form that can be re-used.

Reply:
This change can be improved in many ways. For instance, sharing the configuration from the API and having it inject into all forms that require it.
From API, I would refrain from doing the "injection" before dry run in every form, and I would add this ability to the centralized form.io dryRun API call.
That said, it was my understanding that we could take the easiest and simple route to finish this ticket to later do an improvement. 
Having the changes exactly the same already helps, and we can have a tech debt to have "File Upload Validations Centralization for UI and API", which can also include the max size for files that is also hard-coded on every form (if I am not wrong).
My suggestion, keep it simple and create a tech debit.

Reply:
I added a hidden field to calculate on server.